src/memgraph.cpp

```diff
@@ -355,6 +356,12 @@ DEFINE_VALIDATED_string(query_modules_directory, "",
                           return true;
                         });
 
+// NOLINTNEXTLINE (cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_string(callable_mappings_path, "",
```

Comment:
@gvolfing, we should rename this flag to `--query-callable-mappings-path` because we have these categories in the flags naming (e.g. storage related flags are prefixed with `storage`, auth-related flags are prefixed with `auth`)